docs/en/tutorials/getting-started.rst

```diff
@@ -717,8 +717,8 @@ the bi-directional reference:
     {
         // ... (previous code)
 
-        private $reportedBugs = null;
-        private $assignedBugs = null;
+        protected $reportedBugs = null;
```

Comment:
Could you explain why? All other instance variables are marked as `protected`. Why should these two be `private`?

More importantly, they were declared as `protected` when first added to the `User` class. If there is a legitimate need for the change to the visibility, let me know and I'll incorporate it into the section where the visibility changes.